query-input/src/main/java/de/brandwatch/minianalytics/queryinput/solr/config/SolrConfig.java

```diff
@@ -0,0 +1,31 @@
+package de.brandwatch.minianalytics.queryinput.solr.config;
+
+import org.apache.solr.client.solrj.SolrClient;
+
+import org.apache.solr.client.solrj.impl.HttpSolrClient;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.solr.core.SolrTemplate;
+import org.springframework.data.solr.repository.config.EnableSolrRepositories;
+
+@Configuration
+@EnableSolrRepositories(
+        basePackages = "de.brandwatch.minianalytics.queryinput.solr.repository"
+)
+public class SolrConfig {
+
+    @Value("${spring.data.solr.host}")
+    String solrURL;
+
+    @Bean
+    public SolrClient solrClient() {
```

Comment:
As solrURL is only used here, it is better to add the `@Value` as parameter to this method:
```java
public SolrClient solrClient(@Value("${spring.data.solr.host}") String solrURL) {
```